src/components/post/index.js

```diff
@@ -62,6 +62,79 @@ const Post = ({
     }
   };
 
+  let likeIcon = null;
+  if (isLiked) {
+    likeIcon = <LikeRed />;
+  } else {
+    likeIcon = <Like />;
+  }
+
+    const showCommentModal = () => {
+        setModal('Add a comment...', 
+        <AddCommentModal 
+            setTriggerUpdate = {setTriggerUpdate}
+            id={id}
+            currentUserName={currentUserName}
+            currentUserInitials={currentUserInitials}
+            setUpdateComments={setUpdateComments}
+        />)
+        openModal()
+        }
+
+    const findComments = async () => {
+          const tempArray = [];
+          const res = await get(`posts/${id}/comments`);
+          const tempComments = res.data.comments;
+          await Promise.all(
+            tempComments.map(async (comment) => {
```

Comment:
Why does this the map callback need to be async?

Reply:
Was left there from a previous version of that function, has been removed.